catalog/nsili/catalog-nsili-endpoint/src/main/java/com/connexta/alliance/nsili/endpoint/managers/ProductMgrImpl.java

```diff
@@ -54,17 +55,20 @@
     @Override
     public GetParametersRequest get_parameters(Product prod, String[] desired_parameters,
             NameValue[] properties) throws ProcessingFault, InvalidInputParameter, SystemFault {
+
+        String id = UUID.randomUUID().toString();
+
         GetParametersRequestImpl getParametersRequest = new GetParametersRequestImpl();
 
         try {
-            _poa().activate_object_with_id("get_parameters".getBytes(Charset.forName(ENCODING)),
+            _poa().activate_object_with_id(id.getBytes(Charset.forName(ENCODING)),
                     getParametersRequest);
         } catch (ServantAlreadyActive | ObjectAlreadyActive | WrongPolicy e) {
-            LOGGER.warn("get_parameters : Unable to activate getParametersRequest object.");
+            LOGGER.warn("get_parameters : Unable to activate getParametersRequest object. {}", e.getLocalizedMessage());
```

Comment:
We can just throw e here, instead of getting the localized message. Also recommend changing this to error level